Gemfile

```diff
@@ -17,6 +17,7 @@ gem 'lograge', '~> 0.7.1'
 gem 'pg', '~> 0.21.0'
 gem 'puma', '~> 3.11'
 gem 'rails', '~> 5.1', '>= 5.1.4'
+gem 'roo', '~> 2.7.1', require: false
```

Comment:
Can you add this to `LICENSE.md`?

Reply:
Gemfile.lock adds a dependency for roo (rubyzip). Should I include that in the LICENSE file as well? Do we include other dependencies there?